sdk/communication/communication-common/src/autoRefreshTokenCredential.ts

```diff
@@ -42,11 +50,20 @@ export class AutoRefreshTokenCredential implements TokenCredential {
   private disposed = false;
 
   constructor(refreshArgs: CommunicationTokenRefreshOptions) {
-    const { tokenRefresher, token, refreshProactively } = refreshArgs;
+    const {
+      tokenRefresher,
+      token,
+      refreshProactively,
+      refreshTimeBeforeTokenExpiryInSeconds
```

Comment:
please rename all occurrences of `refreshTimeBeforeTokenExpiryInSeconds` to `refreshIntervalBeforeTokenExpiryInSeconds`